ui/app/css/itcss/components/popover.scss

```diff
@@ -0,0 +1,328 @@
+
```

Comment:
Are you familiar with BEM css style? We try to mostly follow it throughout the codebase, and something like it will be important as we build out a design system. Intro and further links are here: http://getbem.com/introduction/

I think it is best to change the naming here to reflect that. (Sorry, I know this is an annoying request)